projects/finder-frontend/docker-compose.yml

```diff
@@ -30,9 +30,17 @@ services:
       - search-api-app
       - memcached
       - nginx-proxy
+      - govuk-account-manager-prototype-app
     environment:
       ASSET_HOST: finder-frontend.dev.gov.uk
       VIRTUAL_HOST: finder-frontend.dev.gov.uk
+      FEATURE_FLAG_ACCOUNTS: "enabled"
```

Comment:
I agree at point of merge, we should remove this before it goes in.
but for the moment there's no point running accounts without the checker, it's the only app that integrates, and I intended this branch as a conveniece for folks developing on it there?

:crossed_fingers: the repos will be public soon and we can add the accounts as a first class citizen.
I'll write some docs to go in with this too about what you might want to add to an existing service to integrate a GOV.UK account

Reply:
My main concern is that this will activate accounts for anyone who runs the Transition Checker locally.  I wouldn't want this to interfere with any development being done by the transition team.

Reply:
Ok cool, I'm going to drop this commit and move it across to it's own PR,
We can then merge the rest of this and keep this one just for ourselves in development until this is all live!

Reply:
Done, boiled this one down to only add things, not change anything and then created [this over here](https://github.com/alphagov/govuk-docker/pull/417).